app/src/renderer/components/governance/ModalVote.vue

```diff
@@ -147,16 +137,6 @@ export default {
     close() {
```

Comment:
I love the ActionModal idea, I was thinking if you like to extend it even more, the action function(s) of the button(s) could be parametric instead of the close function, and the close action removed from every other page and action, like here `onVote()` ends with `this.close()`, and implemented just in the action modal, she will be the only one to know how to close herself.
So once an action button is clicked, if it succeed (or always we can choose) we call the close.

Reply:
we went through this with @faboweb this Monday and is a bit more complex to decouple completely parent and children, so for now $refs will do the work